src/ONE.Models.CSharp/Common/Configuration/Graphs/GraphConfigurationData.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace ONE.Models.CSharp.Models.Common.Configuration.Graphs
+{
+    public class GraphConfigurationData
+    {
+        public string name { get; set; }
```

Comment:
Property names in here should be capitalized

Reply:
Moved to Imposed